content/index.ja.md

```diff
@@ -0,0 +1,162 @@
+* [WCAG 2.2＜英語＞](https://www.w3.org/TR/WCAG22/) は2023年10月5日に公開されました。
+
+WCAG 2.0、2.1、2.2は「後方互換性」があるように設計されており、WCAG 2.2に適合するコンテンツはWCAG 2.1やWCAG 2.0にも適合します。すべてのバージョンを満たしたい場合は、WCAG 2.2のリソースを使用することができ、わざわざ以前のバージョンを探す必要はありません。
+
+2.0の達成基準はすべて2.1に含まれ、2.1の達成基準はすべて2.2に含まれています（ただし、次の段落で説明する達成基準 4.1.1を除く）。
+* WCAG 2.0 には12のガイドラインがあります。
+* WCAG 2.1 は1つのガイドラインと17の達成基準を追加しており、[What's New in WCAG 2.1＜英語＞](/standards-guidelines/wcag/new-in-21/)で紹介されています。
+* WCAG 2.2 はさらに9つの達成基準を追加しており、[What's New in WCAG 2.2](/standards-guidelines/wcag/new-in-22/)で紹介されています。
+
+いくつかの変更点があり、実際に後方互換性をサポートするために関連文書を更新しています。主な変更点は、WCAG 2.2において、1つの達成基準（4.1.1 構文解析）が廃止されたことです。WCAG 2.1とWCAG 2.0の正誤表に追加された注記は、[WCAG 2 FAQ, 4.1.1 Parsing＜英語＞](/standards-guidelines/wcag/faq/#parsing411)で説明されているように、これに対処しています。WCAG 2.2には、異なる言語に関する注記も含まれています。詳細は、[WCAG 2 FAQ, internationalization＜英語＞](/standards-guidelines/wcag/faq/#i18n22)にあります。
+
+WCAG 2.0、WCAG 2.1、WCAG 2.2はすべて存在している標準です。WCAG 2.2は、WCAG 2.1を非推奨にしたり、WCAG 2.1にとって代わったりするものではありません。そして、WCAG 2.1はWCAG 2.0を非推奨にするものでもなければ、WCAG 2.0にとって代わるものでもありません。ただ、W3CはWCAGの最新版を使用することを推奨しています。
+
+## 翻訳
+
+WCAG 2の公認翻訳と非公式翻訳の一覧が、[WCAG 2 Translations](/standards-guidelines/wcag/translations/)にあります。
+
+## WCAG 2.0 は ISO/IEC 40500である {#iso}
+
+WCAG2.0は、ISO規格：ISO/IEC 40500:2012として承認されています。ISO/IEC 40500は、上記で関連リソースとともに紹介したオリジナルのWCAG 2.0と全く同じものです。
+
+ISO/IEC 40500の内容は、[www.w3.org/TR/WCAG20](https://www.w3.org/TR/WCAG20/) から自由に入手できます。また、[ISO catalogue {% include_cached external.html %}](https://www.iso.org/standard/58625.html)から購入することもできます。
+
+ISO規格としてのWCAG 2.0の利点は、[FAQのISO＜英語＞](/standards-guidelines/wcag/faq/#iso)にまとめられています。W3CとISOプロセスに関する詳細は、[W3C PAS FAQ](https://www.w3.org/2010/04/pasfaq)にあります。
+
+**W3Cは、ISOプロセスを通じてWCAG 2.2を提出する予定です。**
+
+## 誰がWCAGを作成しているのか {#wg}
+
+WCAGの技術文書は、ワールド・ワイド・ウェブ・コンソーシアム([W3C](https://www.w3.org))のウェブ・アクセシビリティ・イニシアティブ([WAI](https://www.w3.org/WAI/))に属するアクセシビリティガイドライン ワーキンググループ([AG WG](/GL/))（旧ウェブコンテンツ アクセシビリティガイドライン ワーキンググループ）によって開発されています。
+
+WAIは、「WCAG 2 達成方法集」と「WCAG 2 解説書」を定期的に更新しています。私たちは[コメント](/standards-guidelines/wcag/commenting/)や[新しい達成方法の提案](https://www.w3.org/WAI/GL/WCAG20/TECHS-SUBMIT/)を歓迎しています。
+
+WCAGやその他のWAIにおける作業への参加のしかたは、[Participating in WAI＜英語＞](/about/participating/)でご紹介しています。
+
+## WCAG 3とその他の情報 {#more}
+
+WCAGは、オーサリングツール アクセシビリティガイドライン（ATAG）やユーザーエージェント アクセシビリティガイドライン（UAAG）を含む一連のアクセシビリティガイドラインの1つです。[Essential Components of Web Accessibility＜英語＞](/fundamentals/components/)では、さまざまなガイドライン間の関係性について説明しています。
+
+以下に関する詳細については、[WCAG 2 FAQ](/standards-guidelines/wcag/faq/)を参照してください：
+
+-   ** [mobile accessibility＜英語＞](/standards-guidelines/wcag/faq/#mobile)**においてWCAG 2がカバーする範囲
+-   **  [documents and software＜英語＞](/standards-guidelines/wcag/faq/#wcag2ict)**へのWCAG 2の適用
+-   その他
+
+W3C アクセシビリティガイドライン3.0（旧称「Sliver」）の初期ドラフトに関する情報は、**[WCAG 3 Introduction](/standards-guidelines/wcag/wcag3-intro/)**をご覧ください。
```

Comment:
@MakotoUeki "Sliver" is a typo that will be fixed very soon in the original resource: sorry for that. The name was "Silver". Would you please translate it into Japanese?